Loop/Managers/Alerts/InAppModalDeviceAlertPresenter.swift

```diff
@@ -50,23 +50,18 @@ public class InAppModalDeviceAlertPresenter: DeviceAlertPresenter {
         }
     }
     
-    public func removePendingAlert(identifier: DeviceAlert.Identifier) {
+    public func retractAlert(identifier: DeviceAlert.Identifier) {
         DispatchQueue.main.async {
             self.alertsPending[identifier]?.0.invalidate()
             self.clearPendingAlert(identifier: identifier)
+            self.removeDeliveredAlert(identifier: identifier, completion: nil)
         }
     }
-    
-    public func removeDeliveredAlert(identifier: DeviceAlert.Identifier) {
-        removeDeliveredAlert(identifier: identifier, completion: nil)
-    }
-    
+        
     // For tests only
```

Comment:
[nit] Is this comment still true? I see `removeDeliveredAlert` used in `retractAlert`.